slam-libraries/viam-cartographer/src/io/color.cc

```diff
@@ -1,5 +1,4 @@
-// This is an Experimental variation of cartographer. It has not yet been
-// integrated into RDK.
+// This is an Experimental integration of cartographer into RDK.
```

Comment:
[q] why is experimental capitalized?

Reply:
Cause that's how it's done in orbslam and while I had the same question as you, I just copy/pasted it. Don't like it, though, and the fact that you pointed it out changed my mind about it. I'll go and fix it for both cartographer and orbslam.

Reply:
I’m guessing it’s because Experimental is the name of the label that should be defined somewhere in Viam documentation and not purely an adjective

Reply:
It's lowercase in the documentation:
* https://docs.viam.com/services/slam/
* https://docs.viam.com/tutorials/slam/

Which is why I think it was probably an artifact that originated from somewhere else (maybe a typo in the ticket description).